Source/Core/src/ca/uqac/lif/synthia/random/RandomString.java

```diff
@@ -37,41 +35,99 @@
 	 */
 	protected Picker<Integer> m_lengthPicker;
 
-	protected RandomInteger m_charValuePicker;
+	protected RandomInteger m_charIndexPicker;
```

Comment:
Instead of a `RandomInteger` refactor this field to the more general `Picker<Integer>`